d2js/js/README.md

```diff
@@ -42,24 +42,59 @@ import { D2 } from '@terrastruct/d2';
 const d2 = new D2();
 
 const result = await d2.compile('x -> y');
-const svg = await d2.render(result.diagram);
+const svg = await d2.render(result.diagram, result.options);
+```
+
+Additional Configuration:
```

Comment:
`Configuring render options`

(leave a comment linking to section for full render options)